kuksa_databroker/databroker/src/glob.rs

```diff
@@ -51,3 +52,113 @@ pub fn to_regex(glob: &str) -> Result<Regex, Error> {
     let re = to_regex_string(glob);
     Regex::new(&re).map_err(|_err| Error::RegexError)
 }
+
+fn has_asterisk_in_middle(input: &str) -> bool {
+    let input = input.trim(); // Remove leading and trailing whitespace
+    if input.starts_with('*') || input.ends_with('*') {
+        return false;
+    }
```

Comment:
Can it not have an "asterisk in the middle" if it starts or ends with an asterisk?

Reply:
It seems that such cases do not appear [here](https://github.com/boschglobal/kuksa.val/blob/feature/wildcard_matching_doc/doc/wildcard_matching.md)